packages/dialtone-mcp-server/rollup.config.js

```diff
@@ -0,0 +1,22 @@
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import typescript from '@rollup/plugin-typescript';
+import json from '@rollup/plugin-json';
+
+export default {
+  input: 'src/index.ts',
+  output: {
+    file: 'build/index.js',
+    format: 'es',
+    banner: '#!/usr/bin/env node',
+  },
+  external: ['@modelcontextprotocol/sdk/server/mcp.js', '@modelcontextprotocol/sdk/server/stdio.js'],
+  plugins: [
+    nodeResolve({
+      preferBuiltins: true,
+    }),
+    json(),
+    typescript({
+      tsconfig: './tsconfig.json',
+    }),
```

Comment:
Don't need to pass the file path here, you only need to pass it if the name is different, it looks for `tsconfig.json` file by default.

```suggestion
    typescript(),
```

https://github.com/rollup/plugins/tree/master/packages/typescript#tsconfig